x-pack/plugins/apm/common/anomaly_detection/apm_ml_job.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { DATAFEED_STATE, JOB_STATE } from '../../../ml/common/constants/states';
```

Comment:
With the changed export I guess this can be imported from `'../../../ml/common'` now?

Reply:
cheers, updated!